### DISCUSSION TITLE
Random crashes (`double free or corruption (fasttop)`) when running lots of Stan targets

### DISCUSSION THREAD

**Discussion by mike-lawrence** [ropensci/targets#396] [Help] [answered]
I'm using targets for SBC, and the make process will occasionally crash with a message saying `double free or corruption (fasttop)`, even with `tar_options_set(error='continue')`. Think this is more likely to be a targets thing, or a Stan thing? Any tips on sleuthing out what's going awry?

**Comment by wlandau** [comment, marked as answer]
`targets` has no C code, so it seems more likely to be coming from another package or CmdStan. Did you get any core dump files? If you do, maybe [analyze them with `gdb`](http://www.brendangregg.com/blog/2016-08-09/gdb-example-ncurses.html)?

**Reply by mike-lawrence** [reply]
I'll do that and report to the Stan folks, thanks! :)

**Reply by wlandau** [reply]
Another thing that occurred to me: it may be on the compilation end, when multiple workers are trying to access the same compiled model and check that it was compiled. In the past, `compile = "copy"` in various `stantargets` functions has worked for me. That way, each worker gets its own temporary compiled copy of the model.
